schemas/com.snowplowanalytics.insights/data_structures_dependencies/jsonschema/1-0-0

```diff
@@ -0,0 +1,49 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for recording schema dependencies",
```

Comment:
It will be great for future 'data-meaning' initiatives if we can use the description to document whether this is meant to validate and event, an entity (context), an API, a configuration, or something else.

Reply:
Agree! I'm going to push with an improved description